### ISSUE TITLE
About the old ballistic... 

### ISSUE THREAD

**Issue by Ali109956** [luckydog7/Funkin-android#1073]
It is indeed possiple to beat the old ballistic on mobile. Here is a screenshot for my score on it, and if you think it's botplay. Botplay dosn't get score nethier does it have an accuracy.

![Screenshot_20210531_164301_com ninjamuffin99 funkin](https://user-images.githubusercontent.com/83115279/120202823-b8866900-c22f-11eb-8f50-f0389e730772.jpg)

**Comment by DzGithub2805** [comment]
YOU ARE A PRO 😎

**Comment by andre039** [comment]
guys i need Help how to fix the sprite moving around and not staying in place when the camera moves

**Comment by andre039** [comment]
guys i need Help how to fix the sprite moving around and not staying in place when the camera moves

**Comment by TankPain** [comment]
Ngl ballistic hard is kinda ez for me

**Comment by Ali109956** [comment]
> Ngl ballistic hard is kinda ez for me

Which one of them?
Old ballistic or Remastered ballistic?

**Comment by Ruv023** [comment]
> It is indeed possiple to beat the old ballistic on mobile. Here is a screenshot for my score on it, and if you think it's botplay. Botplay dosn't get score nethier does it have an accuracy.
> 
> ![Screenshot_20210531_164301_com ninjamuffin99 funkin](https://user-images.githubusercontent.com/83115279/120202823-b8866900-c22f-11eb-8f50-f0389e730772.jpg)

yeah.. The one button thing makes you very talented

**Comment by Ali109956** [comment]
> > It is indeed possiple to beat the old ballistic on mobile. Here is a screenshot for my score on it, and if you think it's botplay. Botplay dosn't get score nethier does it have an accuracy.
> > ![Screenshot_20210531_164301_com ninjamuffin99 funkin](https://user-images.githubusercontent.com/83115279/120202823-b8866900-c22f-11eb-8f50-f0389e730772.jpg)
> 
> yeah.. The one button thing makes you very talented

Which button? T e l l m e.

**Comment by DabKnight** [comment]
I beated the old ballistic on hard aswell!
So i confirm this is possible!

**Comment by LarryFrosty** [comment]
yeah it is possible, i beat ballistic hard like 5 times on mobile already
